mws/resource_mws_ncc_binding.go

```diff
@@ -0,0 +1,51 @@
+package mws
+
+import (
+	"context"
+	"log"
+
+	"github.com/databricks/databricks-sdk-go/service/provisioning"
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceMwsNccBinding() common.Resource {
+	type binding struct {
+		WorkspaceId int64  `json:"workspace_id" tf:"force_new"`
+		NccId       string `json:"network_connectivity_config_id" tf:"force_new"`
+	}
+	s := common.StructToSchema(binding{}, common.NoCustomize)
+	p := common.NewPairSeparatedID("workspace_id", "network_connectivity_config_id", "/")
+	createOrUpdate := func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+		acc, err := c.AccountClient()
+		if err != nil {
+			return err
+		}
+		wait, err := acc.Workspaces.Update(ctx, provisioning.UpdateWorkspaceRequest{
+			NetworkConnectivityConfigId: d.Get("network_connectivity_config_id").(string),
+			WorkspaceId:                 int64(d.Get("workspace_id").(int)),
+		})
+		if err != nil {
+			return err
+		}
+		_, err = wait.Get()
+		if err != nil {
+			return err
+		}
+		p.Pack(d)
+		return nil
+	}
+	return common.Resource{
+		Schema: s,
+		Create: createOrUpdate,
+		Read: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+			log.Print("[WARN] Importing this NCC binding is not enabled, skipping...")
+			return nil
+		},
+		Delete: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+			log.Printf("[WARN] Cannot remove network connectivity config binding, only update is supported.")
```

Comment:
When the resource is changed, this will be deleted and recreated. How will that work?

Reply:
since there is only one API to update ncc of a workspace, I've added update method & test